plugins/com.google.cloud.tools.eclipse.apimanagement.test/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Out of curiosity, are the .classpath, .project and .settings files needed if the project is imported with maven?